app/controllers/concerns/caching_headers.rb

```diff
@@ -21,6 +21,8 @@ def set_cache_control_headers(
 
     request.session_options[:skip] = true # no cookies
 
+    @edge_caching_in_place = true # Used primarily for downstream meta information about whether a path is cached.
```

Comment:
Is this Thread-safe? I think it is because the modules are included and used in instances of the ApplicationController so this would be tied to a particular instance...can someone double check my logic there? Is that correct? I want to make sure we don't start leaking variables.

Reply:
I think we can set instance variables here like we could with any method that sits within a controller. But yeah, maybe it's different with included modules?

... For this particular functionality it shouldn't matter since we only care about this in dev environment, but of course better do it the right way so we don't wrongly rely on this if it is not thread safe for future functionality.